base/grafana/gen_token.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+NAMESPACE=${NAMESPACE:=tools}
+TOKEN="$(oc serviceaccounts new-token grafana-sa -n "$NAMESPACE")"
+THANOS_URL="$(oc get route/thanos-querier -n openshift-monitoring -o jsonpath='{.spec.host}')"
+
+# Write the values to a file for Kustomize to consume
+cat <<EOF > grafana-env-config.env
```

Comment:
maybe add `oc adm policy add-cluster-role-to-user cluster-monitoring-view -z grafana-sa -n "$NAMESPACE"`